src/test/java/com/spotify/futures/CompletableFuturesTest.java

```diff
@@ -429,4 +431,23 @@ public void combine5_exceptional() throws Exception {
     exception.expectCause(isA(IllegalStateException.class));
     getCompleted(future);
   }
+
+  private static <T> Matcher<CompletionStage<T>> completesTo(final T expected) {
+    return completesTo(is(expected));
+  }
+
+  private static <T> Matcher<CompletionStage<T>> completesTo(final Matcher<T> expected) {
+    return new CustomTypeSafeMatcher<CompletionStage<T>>("complete to" + String.valueOf(expected)) {
+      @Override
+      protected boolean matchesSafely(CompletionStage<T> item) {
+        try {
+          final T value = item.toCompletableFuture().get(1, TimeUnit.SECONDS);
+          return expected.matches(value);
+        } catch (Exception ex) {
```

Comment:
Would be nice if `ex` somehow got included in the `Description` so that you could diagnose the error without a debugger.